crates/cxx-qt-lib/src/gui/qfont.cpp

```diff
@@ -11,9 +11,9 @@
 
 // https://code.qt.io/cgit/qt/qtbase.git/tree/src/gui/text/qfont.h?h=v5.15.6-lts-lgpl#n344
 // https://code.qt.io/cgit/qt/qtbase.git/tree/src/gui/text/qfont.h?h=v6.2.4#n323
-assert_alignment_and_size(QFont,
-                          alignof(::std::size_t),
-                          sizeof(::std::size_t) + sizeof(::std::int64_t));
+constexpr static ::std::array<::std::size_t, 2> arr{ sizeof(::std::size_t),
+                                                     sizeof(uint) };
+assert_alignment_and_size(QFont, alignof(::std::size_t), arr, arr.size());
```

Comment:
it should probably be changed to `usize` + `u32` as the members are pointer and uint
it will align to the pointer so regardless of whether the uint is 16 or 32 it should be the same size i think